### PR TITLE
164 UI create a footer

### PR DESCRIPTION
**Pull request by TheGreenOak**

### CODE REVIEW

**Comment by YechiamW3** [comment]
@TheGreenOak make sure you check es-lint warnings before pushing :)
As far as I see all of them are good comments.

**Comment by Aradon267** [comment]
@TheGreenOak merged the 404 PR, and saw the footer layout, looks amazing!

File: cv_next/components/ui/pageFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const Footer: React.FC = () => {
+    return (
+        
+        <footer className='p-4 w-full font-light text-muted-foreground text-primary inset-x-0 bottom-0'>
+            <div className='container columns-4 flex justify-center'>
+                <hr className ="w-11/12 h-1 mx-auto my-4 border-0 rounded md:my-10 dark:bg-slate-500"/>
+                <a href="https://www.instagram.com" target="_blank" className='max-w-24 max-h-24 mx-3 pl-4 py-5 dark:fill-slate-100'>
+                    <svg xmlns="http://www.w3.org/2000/svg" width="45" height="45" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" className="lucide lucide-linkedin"><rect width="20" height="20" x="2" y="2" rx="5" ry="5"/><path d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"/><line x1="17.5" x2="17.51" y1="6.5" y2="6.5"/></svg>
+                </a>
+                <a href="https://www.linkedin.com" target="_blank" className='max-w-24 max-h-24 mx-3 pr-4 py-5 dark:fill-slate-100'>
```
Comment (Aradon267) [inline cv_next/components/ui/pageFooter.tsx:16]: Change the link to magshimim next, same for Instagram

**Comment by TheGreenOak** [comment]
@YechiamW3 @Aradon267 Fixed the eslint stuff, can someone review, please?

File: cv_next/components/ui/pageFooter.tsx
Comment (YechiamW3) [inline cv_next/components/ui/pageFooter.tsx:1]: Should probably be under `/components/layout` and not `/components/ui`

**Comment by YechiamW3** [comment]
@TheGreenOak just the one small change for me and I'll approve

File: cv_next/components/ui/pageFooter.tsx
Comment (Aradon267) [reply cv_next/components/ui/pageFooter.tsx:1]: Alon is abroad on a vacation so I updated it

**Review by Aradon267 — APPROVED** [review]

**Review by YechiamW3 — APPROVED** [review]

**Pull request merged**